### PR TITLE
ARC-175+205: Clarify tagging + indicate what date added field really is

### PR DESCRIPTION
**Pull request by mr2ert**
**IMPORTANT:** A bunch of the changes in this PR currently are from https://github.com/ArchivistProject/archivist-web/pull/37 -- once that gets merged in we'll be able to see the changes for this pr alone more clearly.

screen shot 2017-05-08 at 1 11 14 pm
screen shot 2017-05-08 at 1 11 21 pm
screen shot 2017-05-08 at 1 11 32 pm

### CODE REVIEW

**Comment by mr2ert** [comment]
@tjboswell So I'm currently running into this issue
screen shot 2017-05-08 at 1 12 00 pm
Which I think has something to do with the relative/absolute positioning + z-indexes.  However, I'm not up for more css crap today -- so maybe you can help tomorrow?

**Comment by y0ungy3** [comment]
is this what you want?

question

**Comment by mr2ert** [comment]
@y0ungy3 I want the positioning to be the same as in mine, but appearing over the viewer.

File: src/views/upload/components/upload.js [hunk 1]
```diff
@@ -207,9 +234,8 @@ export default class Upload extends Component {
                         <label htmlFor='file-5'>
                             <figure>
                                 <svg xmlns='http://www.w3.org/2000/svg' width='20' height='17' viewBox='0 0 20 17'>
-                                    <path
-                                        d='M10 0l-5.2 4.9h3.3v5.1h3.8v-5.1h3.3l-5.2-4.9zm9.3 11.5l-3.2-2.1h-2l3.4 2.6h-3.5c-.1 0-.2.1-.2.1l-.8 2.3h-6l-.8-2.2c-.1-.1-.1-.2-.2-.2h-3.6l3.4-2.6h-2l-3.2 2.1c-.4.3-.7 1-.6 1.5l.6 3.1c.1.5.7.9 1.2.9h16.3c.6 0 1.1-.4 1.3-.9l.6-3.1c.1-.5-.2-1.2-.7-1.5z'
-                                    />
+                                  <path d='M9,89a81,81 0 1,1 0,2zm51-14c0-13 1-19 8-26c7-9 18-10 28-8c10,2 22,12 22,26c0,14-11,19-15,22c-3,3-5,6-5,9v22m0,12v16'>
```
Comment (tjboswell) [inline src/views/upload/components/upload.js]: was this intentional? this is deleting the upload file image

**Comment by tjboswell** [comment]
the issue seems to be with how overflow (annoyingly) works: http://stackoverflow.com/a/38793442
imo it's not worth the effort to try and work around this as it could cause issues with other tabs. i would just change the left margin of the tooltip so that it's within the sidebar.

File: src/views/upload/components/upload.js (hunk 1 above)
Comment (mr2ert) [reply src/views/upload/components/upload.js]: No -- fixed it locally but still need to push it up.

**Comment by mr2ert** [comment]
@tjboswell I'm going to make a separate pr to get more tooltips in.  Could you squirrel this?

**Comment by tjboswell** [comment]
:shipit:

**Pull request merged**